README.md

```diff
@@ -205,4 +205,6 @@ Blender: Original by Benjamin Amos
 
 Fried Chicken: https://pixabay.com/en/chicken-fried-leg-piece-poultry-37807/
 
-Chicken: http://www.clipartpanda.com/clipart_images/shared-by-leonnardo00-9075011
\ No newline at end of file
+Chicken: http://www.clipartpanda.com/clipart_images/shared-by-leonnardo00-9075011
+
+Toast: http://piq.codeus.net/picture/55650/toast (edited)(resized)
```

Comment:
I found an alternative link, hopefully it works better

http://orig05.deviantart.net/5aeb/f/2009/106/c/3/toast_slice_by_kawaii_muffin.gif

Earlier I was able to pull it up but now even I can't do the same